CHANGELOG.md

```diff
@@ -1,8 +1,19 @@
-# Change Log
+# CHANGELOG
 All notable changes to this project will be documented in this file based on the [Keep a Changelog](http://keepachangelog.com/) Standard. This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.0.0-beta.1](https://github.com/elastic/ecs/tag/v1.0.0-beta.1) - 2018-11-08
```

Comment:
I would keep a section on top with `#Unreleased` and all the headers inside. The release part is then copied into the release notes on Github.